mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
+  //This function generates a random number between 0-1, multiplies it by 10 and returns the total
```

Comment:
The _de facto_ way of writing JS comments is to place comments (with specific format) before the function / variable declaration body. This has been well adopted across the industry, IDEs and document generating tools. 

P.S. Java and Python which have language specification on how to write code comment.